docs/content/authorization-concepts.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Authorization Concepts
+description: Introduction to Authorization 
+sidebar_position: 1
+slug: /authorization-concepts
+---
+
+import { ProductName, ProductNameFormat, RelatedSection } from '@components/Docs';
+
+# Authorization Concepts 
+
+## Authentication vs Authorization
+
+[**Authentication**](https://auth0.com/intro-to-iam/what-is-authentication/) (or **AuthN**) ensures a user's identity. [**Authorization**](https://en.wikipedia.org/wiki/Authorization) (or **AuthZ**) determines if a user can perform a certain action on a particular resource.
+
+For example, when you log in to Google, Authentication is the process of verifying that your username and password are correct. Authorization is the process of ensuring that you can access a given Google service or feature. [For more information about AuthN vs AuthZ, click here.](https://www.okta.com/identity-101/authentication-vs-authorization/)
+
+## What Is Fine-Grained Authorization?
+
+**Fine-Grained Authorization** (FGA) implies the ability of granting specific users permission to perform certain actions in specific resources.
+
+Well-designed FGA systems allow managing permissions to millions of objects and users that change rapidly as objects are added and access permissions are updated. 
+
+A notable example of fine-grained authorization is Google Drive: access can be granted either to documents or to folders, as well as to individual users or users as a group, and access rights regularly change as new documents are created and shared with specific users or groups.
+
+## What Is Role-Based Access Control?
+
+In [**Role-Based Access Control**](https://en.wikipedia.org/wiki/Role-based_access_control) (RBAC), permissions are assigned to users based on their role in a system. For example, a user needs the `editor` role to edit content. [For more information about RBAC, click here.](https://auth0.com/docs/manage-users/access-control/rbac)
+
+RBAC systems allow you to define users, groups, roles, and permissions, then store them in a centralized location. Applications access that information to make authorization decisions.
+
+## What Is Attribute-Based Access Control?
+
+In [**Attribute-Based Access Control**](https://en.wikipedia.org/wiki/Attribute-based_access_control) (ABAC), permissions are granted based on a set of attributes that a user or resource possesses. For example, a user assigned both `marketing` and `manager` attributes is entitled to publish and delete posts that have a `marketing` attribute. [For more information about ABAC, click here.](https://www.okta.com/blog/2020/09/attribute-based-access-control-abac/)
```

Comment:
Same feedback as above. Lets stay within the content that we own, and reference ABAC within okta domain instead of wiki. Also removing the "click here" final sentence since it is made redundant.

Reply:
Also, again, we shouldn't be bolding text like this. That is reserved for UI text.